src/test/java/com/stripe/model/AccountTest.java

```diff
@@ -21,14 +19,12 @@
 import static org.mockito.Mockito.*;
 
 public class AccountTest extends BaseStripeMockTest {
-	public static final String RESOURCE_ID = "acct_123";
-
 	@Test
 	public void testDeserialize() throws StripeException, IOException {
-		String json = getResourceAsString("account.json");
+		String json = getResourceAsString("/api_fixtures/account.json");
 		Account acc = APIResource.GSON.fromJson(json, Account.class);
 
-		assertEquals("acct_1032D82eZvKYlo2C", acc.getId());
+		assertEquals("acct_1234", acc.getId());
```

Comment:
I don't think it's necessary to check the value of every field (especially if you're using stripe-mock instead of an embedded fixture file). Mostly what we're testing here is that `fromJson` does not raise an exception.

In the other deserialization tests, I used just two asserts: `assertNotNull(resource)` + `assertNotNul(resource.getId())` to check for the unlikely case where deserialization would not raise an exception but return `null`, or return an instance with `null` values in attributes.

For models with submodels (e.g. `AccountPayoutSchedule`), we should create a separate deserialization test for the submodel, e.g. in `model/AccountPayoutScheduleTest.java`:
```java
String accountData = getFixture("/v1/accounts/acct_123");
String data = getDataAt(accountData, "payout_schedule");
AccountPayoutSchedule object = APIResource.GSON.fromJson(data, AccountPayoutSchedule.class);
assertNotNull(object);
assertNotNull(object.getInterval());
```

(cf. `ChargeOutcomeTest.java` in the first commit)

Reply:
That seems pretty painful to maintain for the Account resource and all its sub-resources and various versions.

Reply:
They are separate classes in the code, so they should be tested separately. It will certainly be a bit painful to cover all models (mostly due to the fact that our coverage is pretty spotty right now), but once that's done, maintenance should be minimal.